src/Evolution/Systems/CurvedScalarWave/Worldtube/Triggers/OrbitRadius.cpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/Triggers/OrbitRadius.hpp"
+
+#include <array>
+#include <vector>
+
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp"
+#include "Time/Time.hpp"
+#include "Utilities/Serialization/CharmPupable.hpp"
+
+namespace Triggers {
+
+OrbitRadius::OrbitRadius(const std::vector<double>& radii) : radii_(radii) {}
```

Comment:
Do you need to be worried about what happens if two radii are sufficiently close to each other that within a time step the charge will cross both? Should there be two triggers in that case, or is the one enough?

Reply:
One is completely fine, as I am just using it to adjust the time step. In a normal scenario there will easily be 1000 time steps between two radii.